src/map/pet.cpp

```diff
@@ -655,21 +659,21 @@ int pet_catch_process2(struct map_session_data* sd, int target_id)
 
 	//FIXME: delete taming item here, if this was an item-invoked capture and the item was flagged as delay-consume [ultramage]
 
-	i = search_petDB_index(md->mob_id,PET_CLASS);
+	pet = pet_db(md->mob_id);
 
```

Comment:
if(!pet) return ?;

Reply:
true!

Reply:
Seeing it now again, it's actually wrong since we have comments here that other mobs will display angry emoticons. So we still need to continue the code here.